client/web/src/repo/utils.tsx

```diff
@@ -12,3 +12,8 @@ export const getCanonicalURL = (sourceType: RepositoryType | string, node: GitCo
     isPerforceChangelistMappingEnabled() && isPerforceDepotSource(sourceType) && node.perforceChangelist
         ? node.perforceChangelist.canonicalURL
         : node.canonicalURL
+
+export const getInitialSearchTerm = (repo: string) => {
+    const r = repo.split('/')
+    return r[r.length - 1]
```

Comment:
javascript arrays can be negative indexed, right?

```suggestion
    return r[-1]
```

Reply:
@camdencheek No, you have to control range manually that it is within 0 and slice length - 1 range. You can use negative indexes when you work with operators though (like slice or splice)

Reply:
aah, gotcha. TIL, thanks!

Reply:
This is possible in TypeScript, but it becomes JavaScript incompatible if you take this route. I think it's safer to return `r[r.length - 1]` for that reason.

Reply:
There's `Array.at` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) but that might return undefined if the array is empty.